### PR TITLE
[parallel-hashmap] Update parallel-hashmap version

### PR DESCRIPTION
**Pull request by greg7mdp**
**Describe the pull request**

- What does your PR fix? Fixes #

- Which triplets are supported/not supported? Have you updated the CI baseline?

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

### CODE REVIEW

File: ports/parallel-hashmap/portfile.cmake [hunk 1]
```diff
@@ -4,8 +4,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
```
Comment (LilyWangL) [inline ports/parallel-hashmap/portfile.cmake:4]: `include(vcpkg_common_functions)` is deprecated. Please remove it.

**Review by LilyWangL — CHANGES_REQUESTED** [review]

**Comment by LilyWangL** [comment]
Can you please use command `file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)` to handle copyright?

**Comment by greg7mdp** [comment]
@LilyWangL done hopefully this is what you needed

**Comment by strega-nil** [comment]
Cools, thanks @greg7mdp :)

**Pull request merged**